iceoryx_examples/waitset/README.md

```diff
@@ -1,5 +1,16 @@
 # WaitSet
 
+## Threadsafety
+The WaitSet is **not** threadsafe!
+- It is **not** allowed to attach or detach _Triggerable_
+   classes with `attachEvent` or `detachEvent` when another thread is currently
+   waiting for events with `wait` or `timedWait`.
+- Do **not** call any of the WaitSet methods concurrently.
+
+The _TriggerHandle_ on the other hand is threadsafe! Therefore you are allowed to
+attach/detach a _TriggerHandle_ to a _Triggerable_ while another thread may
+trigger the _TriggerHandle_.
```

Comment:
If you refer to the Thread-Safety chapter I would leave it as a chapter because it is the biggest issue in which developers can run. On the other hand, the comment to the TriggerHandle is not a warning - it is a hint/explanation that the developer has here some thread-safety guarantees.

Reply:
There's dozen of other boxes e.g. tip or info, please check them out.

Reply:
I prefer it as a first section/chapter in the document :)